llama_stack/providers/utils/datasetio/dataset_utils.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the terms described in the LICENSE file in
+# the root directory of this source tree.
+
+def get_dataset_type(module: str) -> str:
```

Comment:
this feel like a general util to get the provider type that can be share across all of llama_stack, could we move it to `utils/common`?

Reply:
Will do. (Tomorrow).

Reply:
@yanxi0830 - done,

Reply:
Please review.